plugins/logs/mask.go

```diff
@@ -254,20 +261,20 @@ func (r maskRule) mkdirp(node map[string]interface{}, path []string, value inter
 	return nil
 }
 
-func resultValueToMaskRules(rv interface{}) ([]*maskRule, error) {
-
+func newMaskRuleSet(rv interface{}, onRuleError func(*maskRule, error)) (*maskRuleSet, error) {
```

Comment:
`onRuleError` looks fine to me; definitely better than creating a cyclic dependency or adding logging into the mask.go file.